### PR TITLE
Add explanations about database duplicate indexes

### PR DESCRIPTION
**Pull request by lpellegr**

### CODE REVIEW

**Comment by lpellegr** [comment]
@ow2-proactive/developers Could two people please review the changes?

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -913,11 +913,37 @@ http://www.postgresql.org[Postgres] or
...
+ProActive Workflows and Scheduling distribution is provided with several
+samples for configuring the Scheduler and Resource Manager to use most
+standard RDBMS.
 You can find these examples in `PROACTIVE_HOME/config/rm/templates/` and
 `PROACTIVE_HOME/config/scheduler/templates/` folders.
 
+==== Indexes
+
+The Scheduler and RM databases rely on Hibernate to create and update the
+database schema automatically on startup. This last is configured for adding
```
Comment (tobwiens) [inline src/docs/admin/ProActiveAdminGuide.adoc]: "This last is" I am not following what you mean with last.

**Comment by tobwiens** [comment]
Looks good :+1:

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Fixed. Thank you Tobias.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -913,9 +913,36 @@ http://www.postgresql.org[Postgres] or
 http://www.microsoft.com/en-us/server-cloud/products/sql-server/[SQL Server] if
 you care about performance or notice a slowdown.
 
-ProActive Workflows and Scheduling distribution is provided with several samples
-for configuring the Scheduler and Resource Manager to use most standard RDBMS.
-You can find these examples in `PROACTIVE_HOME/config/scheduler/` folder.
+ProActive Workflows and Scheduling distribution is provided with several
+samples for configuring the Scheduler and Resource Manager to use most
+standard RDBMS.
+You can find these examples in `PROACTIVE_HOME/config/rm/templates/` and
```
Comment (mbenguig) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Not sure the paths are correct..

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -913,9 +913,36 @@ http://www.postgresql.org[Postgres] or
...
+
+==== Indexes
+
+The Scheduler and RM databases rely on Hibernate to create and update the
+database schema automatically on startup. Hibernate is configured for adding
+indexes on frequently used columns, including columns associated to foreign
+keys. The goal is to provide a default configuration that behaves in a
+satisfactory manner with the different features available in the Scheduler.
+For instance, the Jobs housekeeping feature requires indexes on most foreign
+keys since a Job deletion may imply several delete by cascade.
```
Comment (mbenguig) [inline src/docs/admin/ProActiveAdminGuide.adoc]: several delete by cascade -> cascade delete.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -913,9 +913,36 @@ http://www.postgresql.org[Postgres] or
...
+offer control over this strategy.
+As a consequence, this difference of behaviour, once linked to our default
+configuration for Hibernate, may lead to duplicate indexes (two indexes with
+different names for a same column) that are created for columns in RM and
+Scheduler tables.
+
+In case you are using in production _MySQL_, _MariaDB_ or a similar database
+that automatically adds indexes on foreign keys, it is strongly recommended to
+ask your DBA to identify and delete duplicate indexes created by the database
+system since these last may hurt performances.
```
Comment (mbenguig) [inline src/docs/admin/ProActiveAdminGuide.adoc]: these last may ... -> these may ...

**Comment by mbenguig** [comment]
Ok!.

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 2 above)
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: The documentation references the changes introduced by ow2-proactive/scheduling#2537. Except if you suggest another folder for database templates, the paths should be correct.

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 3 above)
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Applied

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 4 above)
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Fixed

**Pull request merged**